app/controllers/middleware_server_controller.rb

```diff
@@ -142,13 +142,15 @@ def add_jdbc_driver
   end
 
   def jdbc_drivers
-    mw_server = MiddlewareServer.find(params[:server_id])
+    mw_server = MiddlewareServer.find(from_cid(params[:server_id]))
     mw_manager = mw_server.ext_management_system
     drivers = mw_manager.jdbc_drivers(mw_server.feed)
 
     render :json => {
       :status => :success, :data => drivers
     }
+    rescue StandardError => err
```

Comment:
Wrong indent, `rescue` should be aligned under `def`. (And `render` under the other `render`.)

But thanks, looks about right :)